engine/api/src/main/scala/pl/touk/nussknacker/engine/api/SpelExpressionBlacklist.scala

```diff
@@ -0,0 +1,42 @@
+package pl.touk.nussknacker.engine.api
+
+import pl.touk.nussknacker.engine.api.exception.BlacklistedPatternInvocationException
+
+import scala.util.matching.Regex
+
+object SpelExpressionBlacklist {
+
+  val default: SpelExpressionBlacklist = SpelExpressionBlacklist(
+    Map(
+      "spring" -> "(org\\.springframework)".r,
+      "system" -> "(java\\.lang\\.System)".r,
+      "thread" -> "(java\\.lang\\.Thread)".r,
+      "runtime" -> "(java\\.lang\\.Runtime)".r,
+      "invoke" -> "(java\\.lang\\.invoke)".r,
+      "reflect" -> "(java\\.lang\\.reflect)".r,
+      "net" -> "(java\\.net)".r,
+      "io" -> "(java\\.io)".r,
+      "nio" -> "(java\\.nio)".r,
+      "exec" -> "(exec\\()".r
+    ))
+}
+
+case class SpelExpressionBlacklist(blacklistedPatterns: Map[String, Regex]) {
+
+  private def getClassNameFromObject(targetObject: Object): String = {
+    targetObject match {
+      case targetClass: Class[_] => targetClass.getName
+      case _: Object => targetObject.getClass.getName
+    }
+  }
+
+  def blockBlacklisted(targetObject: Object, methodName: String): Unit = {
```

Comment:
I prefer when public methods are above private ones - but its just a note

Reply:
Sounds like a good convention, I will try to implement it from now on, thanks